src/main/java/com/faforever/client/i18n/I18n.java

```diff
@@ -93,15 +95,25 @@ public String get(String key, Object... args) {
   }
 
   public String get(Locale locale, String key, Object... args) {
-    return messageSource.getMessage(key, args, locale);
+    try {
+      return messageSource.getMessage(key, args, locale);
+    } catch (Exception e) {
+      log.error(String.format("Could not load message %s with locale %s defaulting to english", key, locale.toString()), e);
```

Comment:
Slf4j logger never need String.format or MessageFormat. It works out of the box with `{}` as the placeholder

Reply:
It cannot take the variable argument to the string and log the throwable but I can just do string concatenation